easybuild/easyblocks/c/cuda.py

```diff
@@ -247,6 +248,16 @@ def create_wrapper(wrapper_name, wrapper_comp):
         # Also create the lib dir as a symlink
         symlink('lib64', os.path.join(new_stubs_dir, 'lib'), use_abspath_source=False)
 
+        # Remove stubs which are not required as the full library is in /lib64 because this duplication
```

Comment:
```suggestion
        # Remove stubs which are not required as the full library is in $EBROOTCUDA/lib64 because this duplication
```
This threw me a little as I thought you were suggesting a system location